### ISSUE TITLE
[FEATURE REQUEST] Download to phone storage from torrents tab

### ISSUE THREAD

**Issue by Invictaz** [LivingWithHippos/unchained-android#349]
When you select a complete series season on the torrents tab it passes to the downloads tab, and thus is downloaded to Real-Debrid and is split up in all separate episodes.

This is for 100+ episodes quite intensive to fish out of other non related downloads. 

Only on the download tab, you can select files to download stuff to your phone to keep forever.

It would be better to have the option to download to phone storage via the torrents tab.

**Comment by LivingWithHippos** [comment]
Yeah since there have been multiple requests to add features of the downloads sections to the torrent tab, I think I'll update the torrent section to be able to do everything
